providers/base/units/led/jobs.pxu

```diff
@@ -349,6 +349,23 @@ requires: manifest.has_led_indicator == 'True'
 command:
     led_control_test.py -n {path} -d 10
 
+unit: template
+template-resource: led-indicator/sysfs-leds
+template-unit: job
+id: led-indicator/sysfs-leds-{name}-auto
+category_id: led
+_summary: Automated check control of {name} LED.
+_description:
+  Automated check that {name} LED turns on and off.
+plugin: shell
+user: root
+flags: also-after-suspend
+estimated_duration: 10
+imports: from com.canonical.plainbox import manifest
+requires: manifest.has_led_indicator == 'True'
+command:
+    led_control_test.py -n {path} -d 10
+
```

Comment:
The job doesn't check if LED turns on and off.

Reply:
Hi @kissiel,
Yes, you are correct, this job is not able to tell if the LED actually is on or off. However, the script checks whether the value is expected or not after writing the value to the node. Wondering if that is good enough.
https://github.com/canonical/checkbox/blob/main/providers/base/bin/led_control_test.py#L61